EIPS/eip-1014.md

```diff
@@ -12,6 +12,8 @@ created: 2018-04-20
 
 Adds a new opcode at 0xf5, which takes 4 stack arguments: endowment, memory_start, memory_length, salt. Behaves identically to CREATE, except using `sha3(msg.sender ++ salt ++ init_code)[12:]` instead of the usual sender-and-nonce-hash as the address where the contract is initialized at.
 
```

Comment:
Here still the wrong formula is cited.

It would also be good to have a short explanation of the difference between ``init_code`` and ``code`` added to the EIP.